src/routes/oath.ts

```diff
@@ -3,11 +3,15 @@ require('dotenv').config();
 import axios from 'axios';
 const querystring = require('query-string');
 
+import {AddTokensToDatabase} from "../database"
+
+import {PollTokensFromDatabase} from "../database";
+import {hubspotUpdatesController} from "../integration/hubspotUpdatesController";
+
+
 const router = Router();
+ let current_user:string;
```

Comment:
Current user is a global variable that is shared between all users that do the oauth flow. So it will get over written with multiple users using the integration at once

Reply:
Thank you, David, I'll look into a solution.

Reply:
Never mind. I think this is a good fix for now until we refactor